control_plane/roles/webui_awx/vars/main.yml

```diff
@@ -12,58 +12,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
-
-# vars file for webui_awx
-
-# Usage: clone_awx.yml
-awx_git_repo: "https://github.com/ansible/awx.git"
-docker_volume: "/var/lib/docker/volumes/{{ docker_volume_name }}"
-awx_repo_path: "{{ docker_volume }}/awx/"
-awx_installer_path: "{{ awx_repo_path }}/installer/"
+# vars file for web_ui
```

Comment:
Have we decided to move back to `web_ui` for the name, or should we keep `webui_awx`? @sujit-jadhav can you confirm?

Reply:
@lwilson , it should be webui_awx.

Reply:
Modified